components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/edit/EditToolbar.kt

```diff
@@ -116,8 +116,12 @@ internal class EditToolbar(
      * Updates the URL. This should only be called if the toolbar is not in editing mode. Otherwise
      * this might override the URL the user is currently typing.
      */
-    fun updateUrl(url: String) {
+    fun updateUrl(url: String, shouldAutoComplete: Boolean = false) {
+        val previousEnabledState = urlView.isEnabled
+
+        urlView.isEnabled = shouldAutoComplete
         urlView.setText(url)
+        urlView.isEnabled = previousEnabledState
```

Comment:
nit: It feels like we could move this even one level lower into `InlineAutoCompleteText`. Maybe by adding an optional `shouldAutoComplete` parameter to `setText()`?

Reply:
I moved it down a level, but that required me modifying the override function to not be overriden since we can't add parameters to the EditText interface (Android native). I'm unsure if this has unexpected results.